docs/views/popup.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Popup
+author: bijington
+ms.author: joverslu
+description: The Popup
+ms.date: 11/23/2021
+---
+
+# Title
```

Comment:
I think something is missing here...

Reply:
I can't tell you how long it took me to work out what was missing 🤣